script.js

```diff
@@ -94,4 +94,18 @@ document.getElementById('add-task-btn').addEventListener('click', function() {
       }
     });
   }
-  
\ No newline at end of file
+  
+  // Password Visibility Toggle
```

Comment:
Remove this comment. It is not needed here, When we can get the working by variable name or by the code, then there is no need to add comments. Whenever you feel that someone is reading your code and it will be hard to understand why you did this, then it is good to add `concise` comments.